1.1-Introduction-to-Schnorr.ipynb

```diff
@@ -90,15 +90,10 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "# Generate a random value in the secp256k1 field\n",
-    "k = random.randrange(1, SECP256K1_ORDER)\n",
-    "print(\"k = {}\\n\".format(k))\n",
-    "\n",
-    "k_key = ECKey().set(k, True)\n",
-    "R = k_key.get_pubkey()\n",
+    "# Generate a random value and its assoctiated curve point. We can use the generate_key_pair convenience function.\n",
+    "k, R = generate_key_pair()\n",
```

Comment:
won't this confuse people because it's technically not a keypair?

Reply:
That's technically true, but it is a scalar/point pair. I think having a convenience function that returns a random nonce/nonce point makes this worthwhile. I could update the comment to say that this isn't technically a key pair?